dataStandard_versioned_docs/version-5/whats-new/api_changes_5.md

```diff
@@ -2,303 +2,314 @@
+    * **Changed** property: [n].programParticipationStatuses[n].statusEndDate (string)
+    * **Changed** property: [n].specialEducationProgramServices[n].serviceBeginDate (string)
+    * **Changed** property: [n].specialEducationProgramServices[n].serviceEndDate (string)
+
+* `POST/ed-fi/studentSpecialEducationProgramAssociations`
+
+  * **Request**
+    * **Added** property: reductionInHoursPerWeekComparedToPeers (number)
+    * **Added** property: shortenedSchoolDayIndicator (boolean)
+    * **Changed** property: beginDate (string)
+    * **Changed** property: endDate (string)
+    * **Changed** property: iepBeginDate (string)
+    * **Changed** property: iepEndDate (string)
+    * **Changed** property: programParticipationStatuses (array)
+    * **Changed** property: specialEducationProgramServices (array)
+    * **Changed** property: programParticipationStatuses[n].statusBeginDate (string)
+    * **Changed** property: programParticipationStatuses[n].statusEndDate (string)
+    * **Changed** property: specialEducationProgramServices[n].serviceBeginDate (string)
+    * **Changed** property: specialEducationProgramServices[n].serviceEndDate (string)
+
+* `GET /ed-fi/studentSpecialEducationProgramAssociations/{id}`
+
+  * **Response**
+    * **Added** property: reductionInHoursPerWeekComparedToPeers (number)
+    * **Added** property: shortenedSchoolDayIndicator (boolean)
+    * **Changed** property: beginDate (string)
+    * **Changed** property: endDate (string)
+    * **Changed** property: iepBeginDate (string)
+    * **Changed** property: iepEndDate (string)
+    * **Changed** property: programParticipationStatuses (array)
+    * **Changed** property: specialEducationProgramServices (array)
+    * **Changed** property: programParticipationStatuses[n].statusBeginDate (string)
+    * **Changed** property: programParticipationStatuses[n].statusEndDate (string)
+    * **Changed** property: specialEducationProgramServices[n].serviceBeginDate (string)
+    * **Changed** property: specialEducationProgramServices[n].serviceEndDate (string)
+
+* `PUT /ed-fi/studentSpecialEducationProgramAssociations/{id}`
+
+  * **Request**
+    * **Added** property: reductionInHoursPerWeekComparedToPeers (number)
+    * **Added** property: shortenedSchoolDayIndicator (boolean)
+    * **Changed** property: beginDate (string)
+    * **Changed** property: endDate (string)
+    * **Changed** property: iepBeginDate (string)
+    * **Changed** property: iepEndDate (string)
+    * **Changed** property: programParticipationStatuses (array)
+    * **Changed** property: specialEducationProgramServices (array)
+    * **Changed** property: programParticipationStatuses[n].statusBeginDate (string)
+    * **Changed** property: programParticipationStatuses[n].statusEndDate (string)
+    * **Changed** property: specialEducationProgramServices[n].serviceBeginDate (string)
+    * **Changed** property: specialEducationProgramServices[n].serviceEndDate (string)
```

Comment:
What changed about these properties? For the dates, was it only the note we have at the bottom of the page? If so, I wonder if we can remove the lines saying that the dates changed?

And for others - `programParticipationStatuses` and `specialEducaitonProgramServices` - can we describe exactly what changed?